receiver/awsxrayreceiver/internal/translator/addtoattrs.go

```diff
@@ -22,3 +22,15 @@ func addInt64(val *int64, attrKey string, attrs pcommon.Map) {
 		attrs.PutInt(attrKey, *val)
 	}
 }
+
+func addStringSlice(val *string, attrKey string, attrs pcommon.Map) {
+	if val != nil {
```

Comment:
It's generally a little easier to follow if we return if val is nil. This will allow the method functionality to be outside of the if statement.

I see other methods in this file follow the format you're using here though, so this isn't required.

Reply:
I agree. I didn't want to make too many unnecessary changes and ended up choosing to follow the same format as the other functions.